client/ayon_blender/api/lib.py

```diff
@@ -164,6 +164,30 @@ def set_app_templates_path():
         os.environ["BLENDER_USER_SCRIPTS"] = app_templates_path
 
 
+def get_container_name(name: str,
+                       namespace: str,
+                       context: Dict,
+                       suffix: str):
+    """Function to get container name
+
+    Args:
+        name: Name of resulting assembly
+        namespace: Namespace under which to host container
+        context: Asset information
+        suffix: Suffix of container
+
+    Returns:
+        The name of the container assembly
+    """
+    node_name = f"{context['folder']['name']}_{name}"
+    if namespace:
+        node_name = f"{namespace}:{node_name}"
+    if suffix:
+        node_name = f"{node_name}_{suffix}"
+
+    return node_name
```

Comment:
I think this function name is confusing, should this be `create_container_name` or `define_container_name`. `get_*` sounds like it's getting it from an existing instance in the scene?

Reply:
Also, I think this should just be a function inside `pipeline.py` since it's very pipeline related logic and not just 'blender libs'